src/Pages/Movies/index.js

```diff
@@ -0,0 +1,5 @@
+const Movies = () => {
+  return <p>Movies</p>;
+};
+
+export default Movies;
```

Comment:
I'd like to suggest *removing Movie component* from Navigation folder. It do not directly relate to navigation. In future, we will create *MovieList* feature instead.